traitsui/examples/demo/Extras/index.rst

```diff
@@ -0,0 +1,2 @@
+These demos showcase some extra, miscellaneous behaviors that are possible
```

Comment:
Nit: the mention of "miscellaneous" here seems to clash with the new folder name "Misc"?